.github/workflows/test.yml

```diff
@@ -28,10 +28,34 @@ jobs:
           pip install pylint
           if [ -f setup.py ]; then pip install -e .; fi
           pip install transformers
+          wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh
+          bash Miniconda3-latest-Linux-x86_64.sh -b -u
```

Comment:
Wait for #104 and use the same conda approach

Reply:
See approach: https://github.com/groq/mlagility/pull/104/commits/cdaea11bd7a0a9944719aba00ab11855368afd8c